### ISSUE TITLE
project.remoteArtifactRepositories is not allowed here

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis#6805]
### How to Reproduce
- Inspect molgenis/pom.xml in IntelliJ

### Expected behavior
- IntelliJ reports no errors

### Observed behavior
- IntelliJ reports error ```project.remoteArtifactRepositories is not allowed here```:
```
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-project-info-reports-plugin</artifactId>
                <version>2.7</version>
                <configuration>
                    <!-- improve report generation performance -->
                    <dependencyLocationsEnabled>false</dependencyLocationsEnabled>
                    <project.remoteArtifactRepositories>enonic</project.remoteArtifactRepositories>
                </configuration>
            </plugin>
```

**Comment by dennishendriksen** [comment]
IntelliJ stops reporting issue after running mvn site once.

**Comment by sidohaakma** [comment]
Need to use remoteRepositories configuration property instead of project.remoteArtifactRepostories. But is crashed on build.
Caused by: java.lang.InstantiationException: org.apache.maven.artifact.repository.ArtifactRepository

You can also define third-party repositories in the repository tag in the pom.xml. Maybe this is a solution?
```
<repositories>
  <repository>
   <id>enonic</id>
   <name>Enonic repository</name>
   <url>http://repo.enonic.com/maven</url>
  </repository>
</repositories> 
```

**Comment by sidohaakma** [comment]
dennishendriksen:

molgenis-security-pom

    <repositories>
        <repository>
            <id>enonic</id>
            <url>http://repo.enonic.com/public/</url>
        </repository>
    </repositories>

**Comment by dennishendriksen** [comment]
Fixed by 54a0850ada4fc0356a8bef7741219c682af3a53f in 7.0
